vit-servicing-station-tests/src/common/data/mod.rs

```diff
@@ -42,3 +44,52 @@ pub fn challenges() -> Vec<Challenge> {
     let funds = gen.funds();
     gen.challenges(&funds)
 }
+
+pub fn valid_fund_snapshot() -> Snapshot {
```

Comment:
Would it be nicer to `impl Arbitrary for Snapshot`?

Reply:
added Arbitrary, however it implies use of quickcheck, so i've limited number of method invocation to 1. Please tell me if you like current setup of tests

Reply:
mmh, I kind of retract my suggestion now 😅 . I guess we already have testing at the lib level https://github.com/danielSanchezQ/vit-servicing-station/blob/b030c1a4ba6881a676d2b2991006af5a814c9478/vit-servicing-station-lib/src/v0/endpoints/proposals/handlers.rs#L106 and that would be the preferred place for property testing I guess, as it is a bit slow here. We can go with a simple test here for now and think of improving test coverage there